### PR TITLE
contrib/intel: expand Jenkins tcp testing

### PR DESCRIPTION
**Pull request by aingerson**
tcp testing defaults to the quick test which only runs a limited
set of tests and very few iterations. Force tcp to run the -t all
testsuite which runs more tests and 10k iterations to catch more
tcp issues.

Signed-off-by: aingerson <alexia.ingerson@intel.com>

### CODE REVIEW

**Comment by nikhilnanal** [comment]
looks good to me.

File: fabtests/test_configs/tcp/all.test [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	av_type: [
+		FI_AV_TABLE,
+	],
+	comp_type: [
+		FT_COMP_QUEUE,
+	],
+	test_class: [
+		FT_CAP_MSG,
+	],
+},
```
Comment (shefty) [inline fabtests/test_configs/tcp/all.test:27]: There's a tcp verify.test which is more complete.

Comment (aingerson) [reply fabtests/test_configs/tcp/all.test:27]: Do you want me to copy it over/expand on it/rename the verify test?

Comment (shefty) [reply fabtests/test_configs/tcp/all.test:27]: Can we just use it directly?  I don't care if it's renamed.

Comment (aingerson) [reply fabtests/test_configs/tcp/all.test:27]: I'll just rename. runfabtests.sh by default looks for an all.test if not provided which is why having an all.test is helpful

**Comment by aingerson** [comment]
Updated

**Review by shefty — APPROVED** [review]
This may be the smallest changes spread across 3 commits ever.

**Comment by shefty** [comment]
Travis timed out for some reason -- restarted, though I don't see how the failure is related.

**Pull request merged**